cmd/server/main.go

```diff
@@ -69,14 +68,25 @@ func main() {
 func handler(s *rainbow.Service, g *garcon.Garcon) http.Handler {
 	r := chi.NewRouter()
 
-	graphiqlHandler, err := graphiql.NewGraphiqlHandler("/graphql")
-	if err != nil {
-		panic(err)
-	}
-
 	r.With(g.JWT.Set).Mount("/", WebHandler(g, *wwwDir))
-	r.With(g.JWT.Chk).Mount("/graphql", apigraphql.Handler(s))
-	r.With(g.JWT.Chk).Mount("/graphiql", graphiqlHandler)
+
+	r.Route("/v0", func(r chi.Router) {
+		h := api.APIHandler{Service: s}
+
+		// HTTP API similar to REST
+		r.HandleFunc("/options", h.Options)
+		r.HandleFunc("/options/{asset}", h.Options)
+		r.HandleFunc("/options/{asset}/{expiry}", h.Options)
+		r.HandleFunc("/options/{asset}/{expiry}/{provider}", h.Options)
+		r.HandleFunc("/options/{asset}/{expiry}/{provider}/{format}", h.Options)
```

Comment:
Always same handler, so why not use a single endpoint, and have parameters using query parameters ?

`/options?asset=xxx&expiry=xxx`

Reply:
In my opinion `/options/BTC/2022-06/Deribit/csv` is more readable and user friendly than the query-string alternative:
`/options?asset=BTC&expiry=2022-06&provider=Deribit&format=csv`

This is my personal opinion, and other may prefer the query-string alternative. This is why the API accepts both.

Do you understand my API design?

Reply:
I think both are OK. I slightly prefer ` /options?asset=BTC&expiry=2022-06&provider=Deribit&format=csv` because I don't want to overthink the order when I am doing a query and it make it easier to combine (like query BTC & ETH if it ever make sense to do but not sure about that)

Reply:
API design:

* All parameters are optional: absence means ALL
* Parameters can be provided in:
	* the URL path (GET and POST)
	* the query-string (GET and POST)
	* the request body (POST only)